pkg/Microsoft.ML.Datasets/Microsoft.ML.Datasets.nupkgproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="Pack">
```

Comment:
I think we'd probably want to use a different nuget package name than `Microsoft.ML.Datasets`. That seems like it could be a useful package name in the future that we may not want to take here.  Maybe adding `Test` or `Sample` or some other such word would help users realize that this NuGet package isn't for typical consumption.